src/parallax/fourth/sections/one.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./one.module.scss";
+import btnStyles from "../button.module.scss";
+
+const Section1 = () => {
+  return (
+    <section className={styles.one}>
+      <div className={styles.left}>
+        <h1>
+          Air <br />
+          balloon <br />
+          flights
+        </h1>
+        <p>
+          Some text saying something about how awesome air balloon flights are
+          and that you should try it sometime, or maybe something else idk
+        </p>
+        <button className={btnStyles.btn}>Order now!</button>
+      </div>
+      <div className={styles.right}>
+        <img src="/assets/p1/air-balloon1.png" />
```

Comment:
todo: Again the `alt` missing!

Suggestion: Have you ever tried [`axe-core/react`](https://www.npmjs.com/package/@axe-core/react)?

Reply:
interesting n very useful package 🧐

Reply:
yes!